cmd/dbufstat/dbufstat

```diff
@@ -343,7 +343,7 @@ def get_compstring(c):
                     "ZIO_COMPRESS_GZIP_6",  "ZIO_COMPRESS_GZIP_7",
                     "ZIO_COMPRESS_GZIP_8",  "ZIO_COMPRESS_GZIP_9",
                     "ZIO_COMPRESS_ZLE",     "ZIO_COMPRESS_LZ4",
-                    "ZIO_COMPRESS_FUNCTION"]
+                    "ZIO_COMPRESS_ZSTD",    "ZIO_COMPRESS_FUNCTION"]
```

Comment:
why wasn't this placed at the end?

Reply:
https://github.com/zfsonlinux/zfs/pull/8044#discussion_r295772607

Reply:
to repeat it again ZIO_COMPRESS_FUNCTION is the end marker for the enum. it must be always placed at the end

Reply:
`to repeat it again`
Might I suggest a slightly less agressive attitude if you want your PR to be merged any time soon? Part of the slow implementation into ZFSonLinux seems to be your attitude at the moment.